src/v/config/configuration.cc

```diff
@@ -779,6 +779,12 @@ configuration::configuration()
       "Update frequency for kafka queue depth control.",
       required::no,
       7s)
+  , zstd_decompress_workspace_bytes(
+      *this,
+      "zstd_decompress_workspace_bytes",
+      "Size of the zstd decompression workspace",
+      required::no,
+      6_MiB)
```

Comment:
it's hard coded as 2MB no?

Reply:
no, 2mb is used in an unlikely branch to handle our tests etc...